modeshape-persistence-tests/src/test/resources/data/cars.xml

```diff
@@ -0,0 +1,46 @@
+					<Nissan_x0020_Altima jcr:primaryType="car:Car" car:maker="Nissan" car:mpgCity="23" car:year="2008" car:msrp="$18,260" car:mpgHighway="32"  jcr:mixinTypes="mix:referenceable" car:model="Altima"/>
+				</Hybrid>
+				<Sports jcr:primaryType="nt:unstructured"  jcr:mixinTypes="mix:referenceable">
+					<Aston_x0020_Martin_x0020_DB9 jcr:primaryType="car:Car" car:engine="5,935_x0020_cc_x0020_5.9_x0020_liters_x0020_V_x0020_12" car:year="2008" car:wheelbaseInInches="108.0" car:msrp="$171,600" car:mpgHighway="19"  car:model="DB9" car:maker="Aston_x0020_Martin" car:mpgCity="12" jcr:mixinTypes="mix:referenceable" car:userRating="5" car:lengthInInches="185.5"/>
+					<Infiniti_x0020_G37 jcr:primaryType="car:Car" car:year="2008" car:msrp="$34,900" car:mpgHighway="24"  car:model="G37" car:valueRating="4" car:maker="Infiniti" car:mpgCity="18" jcr:mixinTypes="mix:referenceable" car:userRating="3"/>
+				</Sports>
+				<Luxury jcr:primaryType="nt:unstructured"  jcr:mixinTypes="mix:referenceable">
+					<Cadillac_x0020_DTS jcr:primaryType="car:Car" car:maker="Cadillac" car:engine="3.6_x0020_liter_x0020_V6" car:year="2008"  jcr:mixinTypes="mix:referenceable" car:userRating="1" car:model="DTS"/>
+					<Bentley_x0020_Continental jcr:primaryType="car:Car" car:maker="Bentley" car:mpgCity="10" car:year="2008" car:msrp="$170,990" car:mpgHighway="17"  jcr:mixinTypes="mix:referenceable" car:model="Continental"/>
+					<Lexus_x0020_IS350 jcr:primaryType="car:Car" car:year="2008" car:msrp="$36,305" car:mpgHighway="25"  car:model="IS350" car:valueRating="5" car:maker="Lexus" car:mpgCity="18" jcr:mixinTypes="mix:referenceable" car:userRating="4"/>
+				</Luxury>
+				<Utility jcr:primaryType="nt:unstructured"  jcr:mixinTypes="mix:referenceable">
+					<Land_x0020_Rover_x0020_LR2 jcr:primaryType="car:Car" car:year="2008" car:msrp="$33,985" car:mpgHighway="23"  car:model="LR2" car:valueRating="5" car:maker="Land_x0020_Rover" car:mpgCity="16" jcr:mixinTypes="mix:referenceable" car:userRating="4"/>
+					<Land_x0020_Rover_x0020_LR3 jcr:primaryType="car:Car" car:year="2008" car:msrp="$48,525" car:mpgHighway="17"  car:model="LR3" car:valueRating="2" car:maker="Land_x0020_Rover" car:mpgCity="12" jcr:mixinTypes="mix:referenceable" car:userRating="5"/>
+					<Hummer_x0020_H3 jcr:primaryType="car:Car" car:year="2008" car:msrp="$30,595" car:mpgHighway="16"  car:model="H3" car:valueRating="4" car:maker="Hummer" car:mpgCity="13" jcr:mixinTypes="mix:referenceable" car:userRating="3"/>
+					<Ford_x0020_F-150 jcr:primaryType="car:Car" car:year="2008" car:msrp="$23,910" car:mpgHighway="20"  car:model="F-150" car:valueRating="1" car:maker="Ford" car:mpgCity="14" jcr:mixinTypes="mix:referenceable" car:userRating="5"/>
+				</Utility>
+			</Cars>
+		</b>
+	</a>
+	<!-- jcr:system jcr:primaryType="mode:system">
```

Comment:
I understand that about the first part of the file; but I'm referring to the commented out around `jcr:versionStorage` which isn't supported anyway